src/NuGet/Microsoft.NETCore.Compilers.nuspec

```diff
@@ -0,0 +1,45 @@
+<?xml version="1.0"?>
+<package xmlns="http://schemas.microsoft.com/packaging/2011/08/nuspec.xsd">
+  <metadata>
+    <id>Microsoft.NETCore.Compilers</id>
+    <description>
+      CoreCLR-compatible versions of the C# and VB compilers for use in MSBuild.
+    </description>
+    <language>en-US</language>
+    <requireLicenseAcceptance>true</requireLicenseAcceptance>
+    <version>$version$</version>
+    <authors>$authors$</authors>
+    <licenseUrl>$licenseUrl$</licenseUrl>
+    <projectUrl>$projectUrl$</projectUrl>
+    <tags>$tags$</tags>
+    <serviceable>true</serviceable>
+    <developmentDependency>true</developmentDependency>
+  </metadata>
+  <files>
+    <file src="$additionalFilesPath$/Microsoft.NETCore.Compilers.props" target="build" />
+    <file src="Dlls/CodeAnalysis/Microsoft.CodeAnalysis.dll" target="tools" />
+    <file src="Dlls/CSharpCodeAnalysis/Microsoft.CodeAnalysis.CSharp.dll" target="tools" />
+    <file src="Exes/CscCore/publish/csc.dll" target="tools" />
```

Comment:
Please add csi.dll, csi.rsp, Microsoft.CodeAnalysis.Scripting and Microsoft.CodeAnalysis.Scripting.CSharp as well.  
Our desktop FX package contains them.